src/test/java/posters/pageobjects/pages/checkout/BillingAddressPage.java

```diff
@@ -21,9 +21,10 @@
 
     @Override
     @Step("ensure this is a billing address page")
-    public void isExpectedPage()
+    public BillingAddressPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.